src/index.html

```diff
@@ -3,10 +3,388 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@200;300;400;500;600;700;800&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo-image"
+          class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="#apple" class="nav__list-link nav__list-link--is-active is-active">Apple</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#samsung" class="nav__list-link">Samsung</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#smartphones" class="nav__list-link">Smartphones</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#laptops&computers"
+              class="nav__list-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#gadgets" class="nav__list-link">Gadgets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#tablets" class="nav__list-link">Tablets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#photo" class="nav__list-link">Photo</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#video" class="nav__list-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="grid">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac 27&quot; Retina 5K Monoblock"
+            class="card__image"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews-count">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
```

Comment:
It's hard to read now. Fix it everywhere 
```suggestion
            </div>
            
            <p class="card__reviews-count">
              Reviews: 5
            </p>
          </div>
          
          <div class="card__price">
```